### ISSUE TITLE
Train the model on our own dataset !

### ISSUE THREAD

**Issue by obendidi** [shekkizh/FCN.tensorflow#22]
Hello , I've been trying to train the model on my own dataset , I've formatted my annotation to gray scale so that it's the same format as MIT , but when I launch the training , I get a training loss of nan for all iterations, is there something I'm doing wrong , or something I didn't take into consideration ?
Thank you

**Comment by shekkizh** [comment]
Hi @bendidi Nan problem was previously discussed as issue #14 and #17 . 
The problem is how the NUM_OF_CLASSES and annotation values are assigned. Do the solutions posted in the referenced issues solve your problem?

**Comment by obendidi** [comment]
Thanks @shekkizh  I've got the problem solved , it was just an issue with num_classes as you've said and annotation labels

**Comment by SoulGuedria** [comment]
Hi @bendidi 

How can i change the annotation labels of own dataset please ? Thanks a lot :)
